awsiam/iam_user.go

```diff
@@ -155,15 +156,24 @@ func (i *IAMUser) DeleteAccessKey(userName, accessKeyID string) error {
 	return nil
 }
 
-func (i *IAMUser) CreatePolicy(policyName, iamPath, policyTemplate, resource string) (string, error) {
-	tmpl, err := template.New("policy").Parse(policyTemplate)
+func (i *IAMUser) CreatePolicy(policyName, iamPath, policyTemplate string, resources []string) (string, error) {
+	tmpl, err := template.New("policy").Funcs(template.FuncMap{
+		"resources": func(suffix string) string {
+			resourcePaths := make([]string, len(resources))
+			for idx, resource := range resources {
+				resourcePaths[idx] = resource + suffix
+			}
+			marshaled, _ := json.Marshal(resourcePaths)
+			return string(marshaled)
+		},
+	}).Parse(policyTemplate)
 	if err != nil {
 		i.logger.Error("aws-iam-error", err)
 		return "", err
 	}
 	policy := bytes.Buffer{}
 	err = tmpl.Execute(&policy, map[string]interface{}{
-		"Resource": resource,
```

Comment:
We might want to still set resource to the first item in the slice for backwards compatibility?